tests/unit/test_convert.py

```diff
@@ -13,3 +13,17 @@ def test_load_xlsm():
     assert block["line-number"] == 1
     assert "OrganisationURI" in block["line"]
     assert log.mime_type == "application/vnd.ms-excel"
+
+
+def test_converted_csv_saved():
```

Comment:
This test only covers .xlsm, if you tried .geojson or .gpkg. This only works when a reader exists. It will also copy a csv from one place to another when we don't need it to.